### PR TITLE
Kill Failed Validation Enum Member

### PR DESCRIPTION
**Pull request by Bwc9876**
Idk clay asked me to

### CODE REVIEW

File: NewHorizons/Schemas/body_schema.json [hunk 1]
```diff
@@ -2047,13 +2047,11 @@
       "description": "",
       "x-enumNames": [
         "Sockets",
-        "States",
-        "FailedValidation"
+        "States"
```
Comment (FreezeDriedMangos) [inline NewHorizons/Schemas/body_schema.json:2050]: wait why is States duplicated now

File: NewHorizons/Schemas/body_schema.json [hunk 2]
```diff
@@ -2047,13 +2047,11 @@
...
       ],
       "enum": [
         "sockets",
-        "states",
-        "FailedValidation"
+        "states"
```
Comment (FreezeDriedMangos) [inline NewHorizons/Schemas/body_schema.json:2054]: dupe again

**Review by FreezeDriedMangos — CHANGES_REQUESTED** [review]

File: NewHorizons/Schemas/body_schema.json (hunk 2 above)
Comment (Bwc9876) [reply NewHorizons/Schemas/body_schema.json:2054]: Not duplicated, it was just the dif rendering that it was removed and re-added

**Review by FreezeDriedMangos — APPROVED** [review]
awesome thank you

**Comment by JohnCorby** [comment]
does the [JsonIgnore] thing not work?

**Pull request merged**